src/Files.App/Filesystem/FilesystemOperations/Helpers/FilesystemHelpers.cs

```diff
@@ -737,35 +710,13 @@ private static async Task<(List<FileNameConflictResolveOptionType> collisions, b
 
 		public static bool HasDraggedStorageItems(DataPackageView packageView)
 		{
-			return packageView is not null && packageView.Contains(StandardDataFormats.StorageItems);
-		}
-
-		public static async Task<bool> CheckDragNeedsFulltrust(DataPackageView packageView)
-		{
-			if (packageView.Contains(StandardDataFormats.StorageItems))
-			{
-				try
-				{
-					_ = await packageView.GetStorageItemsAsync();
-					return false;
-				}
-				catch (Exception ex) when ((uint)ex.HResult == 0x80040064 || (uint)ex.HResult == 0x8004006A)
-				{
-					return true;
-				}
-				catch (Exception ex)
-				{
-					App.Logger.Warn(ex, ex.Message);
-					return false;
-				}
-			}
-			return false;
+			return packageView is not null && (packageView.Contains(StandardDataFormats.StorageItems) || packageView.Contains("FileDrop"));
```

Comment:
accidentally merged `packageView.Contains("FileDrop")` back in?

Reply:
On purpose, this is the real FileDrop.